guides/common/modules/proc_adding-a-default-http-proxy.adoc

```diff
@@ -36,12 +38,18 @@ $ hammer http-proxy create \
 --username=_My_HTTP_Proxy_User_Name_ \
 --password=_My_HTTP_Proxy_Password_ \
 --url http://_http-proxy.example.com_:8080
+--content_default_http_proxy
```

Comment:
Just a small change in command, and not sure if we should include `--organization "My_Organization" --location "My_Location"` here as well
```suggestion
--url http://_http-proxy.example.com_:8080 \
--content-default-http-proxy true
```

Reply:
I'm not sure about org and loc either, but I'd say it's out of scope.